lib/workers/repository/update/pr/index.ts

```diff
@@ -249,7 +272,9 @@ export async function ensurePr(
     }
   }
 
-  const prBody = await getPrBody(config);
+  const prBody = await getPrBody(config, {
+    debugData: existingPr?.bodyStruct?.debugData,
```

Comment:
Let's call it here rather than inside of `getPrBody` call:

```suggestion
    debugData: updatePrDebugData(existingPr?.bodyStruct?.debugData),
```